package.json

```diff
@@ -12,12 +12,11 @@
   },
   "dependencies": {
     "@openzeppelin/contracts": "5.0.0",
-    "@prb/math": "4.0.2"
+    "@prb/math": "github:PaulRBerg/prb-math"
```

Comment:
Let's please add a specific commit hash so that we don't accidentally pull an updated version with breaking changes.

Reply:
OK.